### PR TITLE
[issue-331] Terraform plan or apply shows changes even if there were …

### PR DESCRIPTION
**Pull request by dondon3345678**
Change external_region_id type to String.

Signed-off-by: Daniel Chuang <tchuang@vmware.com>

### CODE REVIEW

File: vra/resource_flavor_profile.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func resourceFlavorProfile() *schema.Resource {
 				Optional: true,
 			},
 			"external_region_id": {
-				Type:     schema.TypeMap,
+				Type:     schema.TypeString,
```
Comment (Prativa20) [inline vra/resource_flavor_profile.go:37]: Can you verify this in the data source for flavor profile as well?

**Review by Prativa20 — CHANGES_REQUESTED** [review]

File: vra/resource_flavor_profile.go (hunk 1 above)
Comment (dondon3345678) [reply vra/resource_flavor_profile.go:37]: There is no such data source.

Comment (Prativa20) [reply vra/resource_flavor_profile.go:37]: okay, that's a miss. We will have to add one.

**Review by Prativa20 — APPROVED** [review]
looks good!

**Pull request merged**